app/connectors/VatReturnConnector.scala

```diff
@@ -38,6 +39,9 @@ class VatReturnConnector @Inject()(config: Configuration, httpClient: HttpClient
       s"$baseUrl/return",
       coreVatReturn
     )
+  }
 
+  def get(period: Period)(implicit hc: HeaderCarrier): Future[EtmpVatReturn] = { // TODO - HttpParser
```

Comment:
Is the HttpParser needed?

Reply:
Was just thinking or error handling at this point. Do we not need it? I think I've already added it in 167.